src/main/kotlin/io/perfometer/cli/Cli.kt

```diff
@@ -0,0 +1,52 @@
+package io.perfometer.cli
+
+import java.io.File
+import kotlin.script.experimental.annotations.KotlinScript
+import kotlin.script.experimental.api.EvaluationResult
+import kotlin.script.experimental.api.ResultWithDiagnostics
+import kotlin.script.experimental.api.ScriptCompilationConfiguration
+import kotlin.script.experimental.api.defaultImports
+import kotlin.script.experimental.host.toScriptSource
+import kotlin.script.experimental.jvm.dependenciesFromCurrentContext
+import kotlin.script.experimental.jvm.jvm
+import kotlin.script.experimental.jvmhost.BasicJvmScriptingHost
+import kotlin.system.exitProcess
+
+internal object PerfScriptConfiguration : ScriptCompilationConfiguration({
+    defaultImports("io.perfometer.dsl.*")
+    jvm {
+        dependenciesFromCurrentContext(wholeClasspath = true)
+    }
+})
+
+@KotlinScript(fileExtension = "perf.kts", compilationConfiguration = PerfScriptConfiguration::class)
+internal abstract class PerfScript
+
+internal fun evalScenarioFile(scriptFile: File): ResultWithDiagnostics<EvaluationResult> {
+    return BasicJvmScriptingHost().eval(
+        scriptFile.toScriptSource(),
```

Comment:
I find having named parameters helpful in such situations. Also if the eval API changes, named parameters may save us a refactoring.